src/cljs/main/broadfcui/common/filter.cljs

```diff
@@ -17,12 +17,12 @@
 
 
 (defn section [{:keys [title on-clear content]}]
-  [:div {}
+  [:div {:data-test-id (str title "-facet_section")}
```

Comment:
should it be "-facet-section"? not sure if we want to mix dash and underscore. do we do this elsewhere?